### PR TITLE
Bug 1976939: Interacting with CatalogSource page

### PR DESCRIPTION
**Pull request by dtaylor113**
PR adds a 5 minute timeout for `Status==='READY'` check in `beforeEach()` for Admin -> Cluster Settings ->  Configuration
OperatorHub details page to ensure catalog source is READY before each test.  This TC tests for several other detail page fields and values to verify the page; so checking for `Status === READY` isn't neccesary for this particular test, but later tests edit the poll interval where we probably want the catalog source to be `READY`.

Also removed obsolete `/views/catalog-source.view.ts` helper class which is no longer used since [switching to Red Hat Operators](https://github.com/openshift/console/pull/9420) for e2e testing.  We are no longer retreiving Operators from a custom catalog source, so this helper class can be removed.

### CODE REVIEW

**Review by spadgett — CHANGES_REQUESTED** [review]
@dtaylor113 I don't think this will address the problem as later checks like listing package manifests will fail if the catalog source isn't ready.

**Comment by dtaylor113** [comment]
> @dtaylor113 I don't think this will address the problem as later checks like listing package manifests will fail if the catalog source isn't ready.

@spadgett, are you saying the 5 min. timeout isn't enough?

File: frontend/packages/integration-tests-cypress/support/selectors.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ declare global {
       byLegacyTestID(selector: string): Chainable<Element>;
       byButtonText(selector: string): Chainable<Element>;
       byDataID(selector: string): Chainable<Element>;
-      byTestSelector(selector: string): Chainable<Element>;
+      byTestSelector(selector: string, options?: object): Chainable<Element>;
```
Comment (TheRealJon) [inline frontend/packages/integration-tests-cypress/support/selectors.ts:10]: We should be using the same type def that Cypress uses for the `options` arg in `cy.get`

Comment (dtaylor113) [reply frontend/packages/integration-tests-cypress/support/selectors.ts]: updated, thanks

**Comment by spadgett** [comment]
@dtaylor113 Sorry, I misread the description. I thought you had remove the status ready check.

**Review by TheRealJon — APPROVED** [review]
/lgtm
/approve

**Pull request merged**